src/common/index.ts

```diff
@@ -488,6 +488,7 @@ export interface IMenuSettings {
  * the themes to use for the menu and the editor.
  */
 export interface IAppSettings {
+  enableAutostart: boolean;
```

Comment:
It would be cool if you added a comment here, maybe a bit about what the consequences on the different platforms are. We will use the same text in the documetation on the homepage.